src/pages/Profile/Profile.jsx

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-// import person from '../../images/person.png';
-// ^^ blank avatar for profile pic, uncomment the above line to use
+import person from '../../images/person.png';
 import './Profile.css';
 
 const Profile = ({ firstName, lastName, stats }) => {
   return (
     <div className="profile-page">
-      {/* TODO:
-      Complete the profile page here.
-      For the profile picture, use the avatar placeholder image that has
-      already been imported for you on line 3.
-      For the name & stats to display, use the props (firstName, lastName, and stats)
-      that have already been declared. */}
-      {firstName} {lastName} {stats}
+      <div className="avatar">
+        <img src={person} />
+      </div>
+      <div>
+        <div id="name">{firstName} {lastName}</div>
+      </div>
+      <div className="stats">
+        <div id="numbers">{stats.map((s, index) => <div key={index}>{s.number}</div>)}</div>
+        <div id="descriptions">{stats.map((s, index) => <div key={index}>{s.description}</div>)}</div>
+      </div>
```

Comment:
the Figma shows the number and the descriptions are aligned whereas the image attached above shows the number isn't aligned with its corresponding text. the two map function calls can get reduced into one since both `description` and `number` is known given the same map function ( hint to the previous comment :] ).